research-software-directory/RSD-as-a-service
# Split related topics into related software and projects sections

# Related software and related project sections

Closes #983

Changes proposed in this pull request:
* The related topic section on edit software and edit project page is split into 2 separate sections.
* The title "Related tools" on software page is changed to "Related software"   
* The order of related "items" is alphabetic on name/title

How to test:
* `make start` to build app
* login as rsd-admin if you want to manipulate existing items or create new item
* navigate to edit software page, confirm that menu has two separate sections: related software and related projects
* suggestions about used icons, titles and labels are welcome
* navigate to edit project page, confirm that menu has two separate sections: related software and related projects
* suggestions about used icons, titles and labels are welcome

## Example edit software related software section
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/0b19e50a-2795-40e7-8d36-ea7e79b9acef)

## Example edit software related projects section
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/dd6d03ba-8060-46aa-857b-dde53d4412a3)

## Example edit project related projects section
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/22fcadb4-cd41-4409-9b0e-b53de396e191)

## Example edit project related software section
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/be5c18bb-fca9-4d73-8080-348c3ae27762)

## Example relates software title on software page
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/aaffe935-95f2-421a-8f36-c55b8528f1e0)

PR Checklist:
